newsfragments/439.feature.rst

```diff
@@ -0,0 +1,2 @@
+Enable the optimizer by default. The optimizer can still be disabled
+by supplying `--optimize=false` as an argument.
```

Comment:
@mjobuda I added this file to your PR. It ensures your change will be mentioned in the upcoming release notes. I will wait for CI to finish and then merge.